homeassistant/helpers/script.py

```diff
@@ -411,8 +412,17 @@ async def _async_step(self, log_exceptions):
             async with trace_action(self._hass, self, self._stop, self._variables):
                 if self._stop.is_set():
                     return
+
+                action = cv.determine_script_action(self._action)
+
+                if not self._action.get(CONF_ENABLED, True):
```

Comment:
You can move this above the determine script action call.

Reply:
No? It uses that in its log message (if the alias is missing), which is consistent with other log messages.

Reply:
Ah right. Missed that.